### PR TITLE
Version/3.24

### PR DESCRIPTION
**Pull request by lbudai**

### CODE REVIEW

**Comment by lbudai** [comment]
@kira-syslogng : build packages please

File: packaging/rhel/syslog-ng.spec [hunk 1]
```diff
@@ -573,6 +573,9 @@ fi
 %{_datadir}/%{name}/tools/
 
 %changelog
+* Tue 01 Oct 2019 Laszlo Budai <laszlo.budai@balabit.com> - 3.24.1-1
```
Comment (MrAnno) [inline packaging/rhel/syslog-ng.spec]: `error: bad date in %changelog: Tue 01 Oct 2019 Laszlo Budai <laszlo.budai@balabit.com> - 3.24.1-1`

I guess the output of the following Python snippet should be correct: `Tue Oct 01 2019`
```python
time.strftime("%a %b %d %Y", time.localtime(time.time()))
```

Comment (lbudai) [reply packaging/rhel/syslog-ng.spec]: or just `date "+%a %b %d %Y"`

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Kira-dbld-distro-packages: Build FAILURE

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by lbudai** [comment]
@kira-syslogng : build packages please

**Comment by kira-syslogng** [comment]
Kira-dbld-distro-packages: Build SUCCESS

**Pull request merged**